### PR TITLE
CORTX-28870: Adding the allowable list for Alex scan

### PR DESCRIPTION
**Pull request by venkuppu-chn**
Signed-off-by: Venkatesh K <venkatesh.k@seagate.com>

# Problem Statement
Alex scan generate false positive

# Design
We have added some allowable list to ignore the false positives

# Coding
   Checklist for Author
-  [x] Coding conventions are followed and code is consistent

# Testing 
  Checklist for Author
- [ ] Unit and System Tests are added
- [ ] Test Cases cover Happy Path, Non-Happy Path and Scalability
- [ ] Testing was performed with RPM

# Impact Analysis
  Checklist for Author/Reviewer/GateKeeper
- [ ] Interface change (if any) are documented
- [ ] Side effects on other features (deployment/upgrade)
- [ ] Dependencies on other component(s)

# Review Checklist 
  Checklist for Author
- [x] JIRA number/GitHub Issue added to PR
- [x] PR is self reviewed
- [x] Jira and state/status is updated and JIRA is updated with PR link
- [x] Check if the description is clear and explained

# Documentation
  Checklist for Author
- [ ] Changes done to WIKI / Confluence page / Quick Start Guide

### CODE REVIEW

File: .alexrc [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "allow": ["execute", "host-hostess", "remains", "reject", "traps", "invalid", "period", "invalid", "execution", "simple", "failure"]
```
Comment (rahul-tripathi-git) [inline .alexrc:2]: invalid is twice in this list, please check.

**Review by rahul-tripathi-git — CHANGES_REQUESTED** [review]

**Review by cdeshmukh — APPROVED** [review]

File: .alexrc (hunk 1 above)
Comment (venkuppu-chn) [reply .alexrc]: Addressed. Done.

**Review by rahul-tripathi-git — APPROVED** [review]
Looks good to me as well.

**Comment by venkuppu-chn** [comment]
Due to issue in CLA sign, I have created new PR. 

https://github.com/Seagate/cortx-rgw-integration/pull/51

**Pull request closed without merging**